Tests/scripts/configure_tests.py

```diff
@@ -42,7 +42,8 @@
 # File names
 ALL_TESTS = ["scripts/script-CommonIntegration.yml", "scripts/script-CommonIntegrationPython.yml",
              "scripts/script-CommonServer.yml", "scripts/script-CommonServerUserPython.yml",
-             "scripts/script-CommonUserServer.yml", "scripts/CommonServerPython/CommonServerPython.yml"]
+             "scripts/script-CommonUserServer.yml", "scripts/CommonServerPython/CommonServerPython.yml",
```

Comment:
i think `scripts/CommonServerPython/CommonServerPython.yml` is not relevant anymore